make/assets/istio/clusterrole.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I know we already have the `make/assets` directory, but in this case I think it is better to have those files under `make/istio/assets`. I may eventually move the contents of `make/assets` to a more meaningful directory structure.

Reply:
Sure, so I will move to `make/istio/assets` in next patch. Thanks!